src/validation.cpp

```diff
@@ -2234,7 +2234,6 @@ bool CChainState::FlushStateToDisk(
     FlushStateMode mode,
     int nManualPruneHeight)
 {
-    LOCK(cs_main);
```

Comment:
Removed locks are generally replaced with `AssertLockHeld(cs_main);` and the corresponding compiler annotations

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18639#issuecomment-613708604).

Reply:
Please also add the annotation in the header

Reply:
> Please also add the annotation in the header
> 
> EXCLUSIVE_LOCKS_REQUIRED(cs_main)

It is already done: https://github.com/bitcoin/bitcoin/blob/fe187562ef9765942d7c7d801e04d70d294d5aef/src/validation.h#L671-L675